lib/osc.py

```diff
@@ -768,12 +768,93 @@ def createMC(self, mc):
+                                            "domainId": "%s",
+                                            "domainName": "%s",
+                                            "markForDeletion": false,
+                                            "vcId": "%s",
+                                            "virtualizationConnectorName": "%s",
+                                            "virtualizationType": "%s",
+                                            "encapsulationType": "%s"
+                                        }
+                                    ]
+                                }
+                            ''' % (
+                        daid, daname, mcname, mcid, apid, model, swver, vssid, domainId, domainName, vcid, vcname, vctype, encapsulation)
+                else:
+                    body = '''
+                                                    {
+                                                        "id": %s,
+                                                        "name": "%s",
+                                                        "managerConnectorName": "%s",
+                                                        "managerConnectorId": "%s",
+                                                        "applianceSoftwareVersionId": "%s",
+                                                        "applianceModel": "%s",
+                                                        "applianceSoftwareVersionName": "%s",
+                                                        "secretKey": "dummy1234",
+                                                        "markForDeletion": false,
+                                                        "virtualSystem": [
+                                                            {
+                                                                
+                                                                "domainId": "%s",
+                                                                "domainName": "%s",
+                                                                "markForDeletion": false,
+                                                                "vcId": "%s",
+                                                                "virtualizationConnectorName": "%s",
+                                                                "virtualizationType": "%s",
+                                                                "encapsulationType": "%s"
+                                                            }
+                                                        ]
+                                                    }
+                                                ''' % (
+                        daid, daname, mcname, mcid, apid, model, swver, domainId, domainName, vcid, vcname,
+                        vctype, encapsulation)
+                action = 'create Distributed Appliance'
+                url = '/api/server/v1/distributedAppliances/%s' %daid
```

Comment:
Same as setting:
action
url 
PUT or POST 
should be done for createDA as well

Reply:
@ychoma The POST action is already there